jekyll/_cci2/ios-tutorial.md

```diff
@@ -0,0 +1,154 @@
+
+1. Open your Xcode workspace or project. 
+2. Use the scheme selector to open the Manage Schemes dialogue box as shown in the following image.
+![Xcode Scheme Selector](  {{ site.baseurl }}/assets/img/docs/ios-getting-started-scheme-selector.png)
+3. In the manage schemes dialog, select the scheme you wish to build, and ensure that the Shared checkbox is enabled.
+![Manage Schemes Dialogue](  {{ site.baseurl }}/assets/img/docs/ios-getting-started-manage-schemes.png)
+4. Commit and push the schemes.
+
+## Build Phases
+
+In this tutorial, builds are divided into dependencies, test, and deployment. The dependencies phase is for installing any Ruby Gems, CocoaPods, Node Modules or other packages required to build the application. The test phase is for building and testing your project.
+
+For iOS projects, CircleCI generates a command to build and test your project using the `xcodebuild` command line tool. The generated command is similar to:
+
+```
+set -o pipefail &&
+  xcodebuild
+    CODE_SIGNING_REQUIRED=NO
+    CODE_SIGN_IDENTITY=
+    PROVISIONING_PROFILE=
+    -sdk iphonesimulator
+    -destination 'platform=iOS Simulator,OS=9.0,name=iPhone 6'
+    -workspace MyWorkspace.xcworkspace
+    -scheme "My Scheme"
+    clean build test |
+      tee $CIRCLE_ARTIFACTS/xcode_raw.log |
+      xcpretty --color --report junit --output $CIRCLE_TEST_REPORTS/xcode/results.xml
+```
+
+If your project uses React Native, CircleCI automatically runs the `test` script phase from your `package.json`.
+
+## Code Signing
+
+It is possible for CircleCI to automatically inject your code signing certificates and unlock the keychain for your build. To use this automated code signing support for your iOS app, perform the following steps:
+
+1. Export your certificates (p12) by opening the Keychain Access.app, and selecting My Certificates in the left menu.
+![Keychain Access.app with the keychain that contains the keys, and My Certificates selected](  {{ site.baseurl }}/assets/img/docs/ios-getting-started-keychain-access-my-certificates.png)
+A certificate with **iPhone Developer:** or **iPhone Distribution:** appears.
+
+2. Click the certificate, then select File > Export Items from the macOS menu bar ensuring that the file format is Personal Information Exchange (.p12). 
+![The Keychain Access.app file menu with Export Items in a hover state](  {{ site.baseurl }}/assets/img/docs/ios-getting-started-keychain-file-menu.png)
+If the option is not available, check that you selected the private key when you selected the certificate.
+![The Keychain Access.app Export dialogue](  {{ site.baseurl }}/assets/img/docs/ios-getting-started-keychain-export-dialogue.png)
+
+3. It is best practice to enter a certificate export password when it is requested. 
+
+## Adding your certificate
```

Comment:
This no longer applies as using `fastlane match` (new recommended way we will describe in the code signing doc) will take care of this. Should delete this section

Reply:
Or better still, remove contents and link to https://codesigning.guide for now